api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -491,6 +510,13 @@ model NodePoolAutoScaling {
   max?: int32;
 }
 
+scalar DiskEncryptionSetResourceId
+  extends Azure.Core.armResourceIdentifier<[
```

Comment:
What are the implications of this?

Reply:
Does this mean we still need maxLen 285 then? What is the length that that defines?

Reply:
A quick google suggests that there is no official limit. I've seen people suggest a [theoretical max](https://learn.microsoft.com/en-us/answers/questions/1189747/max-length-of-azure-resource-id-and-whether-resour) of:
- ~1000: based on the sum of the max length of the various possible components
- 260:  based on the max path size allowed in Windows.... 

Either way OCP says 285 so I think this is useful